src/main/scala/nl/knaw/dans/easy/stage/fileitem/EasyStageFileItem.scala

```diff
@@ -118,7 +118,7 @@ object EasyStageFileItem {
       cfgContent   <- Try { JSON.createFileCfg(s.datastreamLocation.getOrElse(s.unsetUrl), mime, parent, s.subordinate) }
       _            <- writeJsonCfg(sdoDir, cfgContent)
       title        <- Try {s.title.getOrElse(s.pathInDataset.get.getName)}
-      foxmlContent  = getFileFOXML(title, s.ownerId, mime)
+      foxmlContent  = getFileFOXML(title, s.ownerId.get, mime)
```

Comment:
@janvanmansum have you tested what happens when the `ownerId` is a `None`? Same for [line 134](https://github.com/DANS-KNAW/easy-stage-dataset/pull/91/files#diff-59bc01a526b81110d7d86ac30ce29897R134)...